### PR TITLE
M373 show collect record observers that have been removed from a project. Allow the user to remove them from the collect record.

### PR DESCRIPTION
**Pull request by mmnoo**
[Trello card](https://trello.com/c/VpCjDAgq/373-show-observers-no-longer-in-project-on-su-pages)

### CODE REVIEW

**Comment by mmnoo** [comment]
[show observers no longer in project on SU pages](https://trello.com/c/VpCjDAgq/373-show-observers-no-longer-in-project-on-su-pages)

File: src/App/integrationTests/collectRecords/observersInput.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  // testing tradeoff, we dont directly check that Betsy has been deleted from the record,
+  // but a save will reload updated data, which should be missing Betsy and this is likely a good enough test
+  fireEvent.click(screen.getByRole('button', { name: 'Save' }))
+
+  expect(await screen.findByText('Record saved.'))
+  waitFor(() =>
+    expect(observersRow).toHaveTextContent(
+      'Betsy Craig was an observer on this sample unit but is no longer in this project.',
+    ),
+  )
```
Comment (mmnoo) [inline src/App/integrationTests/collectRecords/observersInput.test.js:46]: I tried to assert on dexie but couldn't get it working with the timebox for tests. This should siffice.

File: src/components/mermaidInputs/InputMuiChipSelectWithLabelAndValidation/InputMuiChipSelectWithLabelAndValidation.js [hunk 2]
```diff
@@ -44,27 +47,31 @@ const InputMuiChipSelectWithLabelAndValidation = ({
...
+      <div>
+        <CustomMuiSelect
+          labelId={`${id}-mui-chip-select-with-label-and-validation`}
+          multiple
+          value={value}
+          onChange={handleCheckboxGroupChange}
+          input={<OutlinedInput id="select-multiple-chip" label="Chip" />}
+          renderValue={(selected) => (selected.length ? chips : language.placeholders.select)}
+          label=""
+          displayEmpty={true}
```
Comment (mmnoo) [inline src/components/mermaidInputs/InputMuiChipSelectWithLabelAndValidation/InputMuiChipSelectWithLabelAndValidation.js:59]: displayEmpty enables us to show `Choose...` when no observers are present

File: src/language.js [hunk 3]
```diff
@@ -290,6 +292,23 @@ const pages = {
       observers: 'Observers',
       transect: 'Transect',
     },
+    observersSelectHelper: 'Select and observer to add',
```
Comment (alanjleonard) [inline src/language.js:295]: Typo. Should be `an` not `and`

File: src/language.js [hunk 4]
```diff
@@ -290,6 +292,23 @@ const pages = {
...
+    getObserverRemovedFromProjectMessage: (userName) => (
+      <>
+        <strong>{userName}</strong> was an observer on this sample unit but is no longer in this
```
Comment (alanjleonard) [inline src/language.js:298]: Update to "...no longer a part of this project."

File: src/components/RemoveObserverModal/RemoveObserverModal.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+const modalLanguage = language.pages.collectRecord.removeObserverModal
+
+const RemoveObserverModal = ({ isOpen, onDismiss, observer, onSubmit }) => {
+  const observerNameToUse = observer ? getObserverNameToUse(observer) : ''
+
+  const footerContent = (
+    <RightFooter>
+      <ButtonSecondary type="button" onClick={onDismiss}>
+        Cancel
```
Comment (alanjleonard) [inline src/components/RemoveObserverModal/RemoveObserverModal.js:18]: Should this be coming from `language.js`?

**Pull request merged**